spaceship/lib/spaceship/spaceauth_runner.rb

```diff
@@ -49,22 +51,28 @@ def run
         cookie.name.start_with?("myacinfo") || cookie.name == "dqsid" || cookie.name.start_with?("DES")
       end
 
-      yaml = cookies.to_yaml.gsub("\n", "\\n")
+      @yaml = cookies.to_yaml.gsub("\n", "\\n")
 
       puts("---")
       puts("")
       puts("Pass the following via the FASTLANE_SESSION environment variable:")
-      puts(yaml.cyan.underline)
+      puts(@yaml.cyan.underline)
       puts("")
       puts("")
       puts("Example:")
-      puts("export FASTLANE_SESSION='#{yaml}'".cyan.underline)
+      puts("export FASTLANE_SESSION='#{@yaml}'".cyan.underline)
 
-      if mac? && Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true)
-        require 'open3'
-        Open3.popen3('pbcopy') { |input, _, _| input << yaml }
-        puts("Successfully copied text into your clipboard 🎨".green)
+      if @copy_to_clipboard || (mac? && Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true))
```

Comment:
Could you change
```ruby
      if @copy_to_clipboard || (mac? && Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true))
        FastlaneCore::Clipboard.copy(content: @yaml)
        puts("Successfully copied the session string into your clipboard 🎨".green)
      end
```
to
```ruby
      if @copy_to_clipboard == false
        puts("Skipped asking to copy the cookie into your clipboard ⏭️".green)
      elsif @copy_to_clipboard || (mac? && Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true))
        FastlaneCore::Clipboard.copy(content: @yaml)
        puts("Successfully copied the session string into your clipboard 🎨".green)
      end
```
That way if you explicitly set copy_to_clipboard to false, the lane won't ask you for your input to copy the cookie into your clipboard. It's useful when creating non-interactive scripts that run on interactive MacOS terminals.
Example:
```ruby
fastlane_session = Spaceship::SpaceauthRunner.new(copy_to_clipboard: false).run.session_string
```

Reply:
Good suggestion! I didn't think anyone would face such scenario. I'll change this 👍

Reply:
Done! 💪